app/model/editions/templates/AmericanEdition.scala

```diff
@@ -56,12 +56,19 @@ object AmericanEdition {
       // Crosswords
       FrontCrosswords -> Daily(),
     ),
-    zoneId = ZoneId.of("Europe/London"),
-    availability = Daily(),
     capiQueryPrefillParams = CapiQueryPrefillParams(
       timeWindowConfig = TimeWindowConfigInDays(
         startOffset = 0,
         endOffset = 0)
+    ),
+    zoneId = ZoneId.of("Europe/London"),
+    availability = Daily(),
+    ophanQueryPrefillParams = Some(OphanQueryPrefillParams(
+      apiKey = s"fronts-editions-${this.getClass.toString}",
+      timeWindowConfig = TimeWindowConfigInDays(
+        startOffset = 0,
+        endOffset = -7
+      ))
     )
```

Comment:
It's a bit confusing for me that in some places we have apiKey with `this.getClass.toString` like `s"fronts-editions-${this.getClass.toString}",` in some with ctry code like: `apiKey = s"fronts-editions-au",`

we will need a separate one for each edition ?

Reply:
I would suggest to have the same method in all